introduction.md

```diff
@@ -0,0 +1 @@
+Hi, my name is Tesla
```

Comment:
Nice!! Right now my favorite path in CodeSchool is Javascript, mainly "ES2015: The Shape of JavaScript to Come", very interesting.